package.json

```diff
@@ -42,68 +42,75 @@
   },
   "dependencies": {
     "chain-function": "^1.0.0",
-    "classnames": "^1.2.0",
-    "invariant": "^2.1.0",
+    "classnames": "^2.2.5",
+    "invariant": "^2.2.1",
     "property-expr": "^1.3.1",
-    "react-input-message": "^0.9.2",
+    "react-input-message": "^0.12.1",
```

Comment:
its not safe to update these, there are breaking changes, I'd jsut leave the none builddeps where they are for now